config/manager/manager.yaml

```diff
@@ -47,10 +47,10 @@ spec:
             periodSeconds: 10
           resources:
             limits:
-              cpu: "600m"
-              memory: "1Gi"
+              cpu: "200m"
+              memory: "80Mi"
             requests:
-              cpu: "300m"
-              memory: "128Mi"
+              cpu: "100m"
+              memory: "50Mi"
```

Comment:
When I look at this service in GCP, it's regular usage sits at ~96MB.
I think a 128Mi request makes sense on memory.